x-pack/plugins/reporting/server/export_types/printable_pdf/create_job/compatibility_shim.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { KibanaRequest } from 'kibana/server';
```

Comment:
This file was restored and re-written from this source: https://github.com/elastic/kibana/blob/57af846/x-pack/plugins/reporting/server/export_types/printable_pdf/create_job/compatibility_shim.js